libs/auth-api-lib/src/lib/resources/resources.service.ts

```diff
@@ -966,7 +966,7 @@ export class ResourcesService {
           offset: offset,
           where: { name: { [Op.iLike]: `%${searchString}%` } },
           order: [['name', 'asc']],
-          include: [ApiScopeGroup],
+          include: [ApiScope],
```

Comment:
We should just return the domains without including the scopes or groups.

Reply:
FIxed.